Signal/src/contact/SystemContactsFetcher.swift

```diff
@@ -145,3 +186,22 @@ class SystemContactsFetcher: NSObject {
     }
 
 }
+
+struct HashableArray<Element: Hashable>: Hashable {
```

Comment:
Just style. The idiomatic way to get this kind of a hash for something is to call `.hash` (or `.hashValue` in swift).